export async function getUsers(){
    try{

    }
    catch(error){
        res.send(error.message)
    }
}